Services/CO.CDP.Organisation.WebApi/Model/Exceptions.cs

```diff
@@ -1,6 +1,8 @@
 namespace CO.CDP.Organisation.WebApi.Model;
 
 public class UnknownOrganisationException(string message, Exception? cause = null) : Exception(message, cause);
+public class UnknownInvitedPersonException(string message, Exception? cause = null) : Exception(message, cause);
+public class EmptyPersonRoleException(string message, Exception? cause = null) : Exception(message, cause);
```

Comment:
Throwing these new exceptions will result in a 500 error. Do we need to map them to 400 errors?

Reply:
mapped